src/main/java/sdk/sample/Creation.java

```diff
@@ -0,0 +1,152 @@
+package sdk.sample;
+
+import com.azure.resourcemanager.netapp.fluent.NetAppManagementClient;
+import com.azure.resourcemanager.netapp.fluent.models.CapacityPoolInner;
+import com.azure.resourcemanager.netapp.fluent.models.NetAppAccountInner;
+import com.azure.resourcemanager.netapp.fluent.models.VolumeInner;
+import sdk.sample.common.CommonSdk;
+import sdk.sample.common.ProjectConfiguration;
+import sdk.sample.common.Utils;
+import sdk.sample.model.ModelCapacityPool;
+import sdk.sample.model.ModelNetAppAccount;
+import sdk.sample.model.ModelVolume;
+
+public class Creation
+{
+    public static void createANFResources(ProjectConfiguration config, NetAppManagementClient anfClient)
+    {
+        /*
+          Creating ANF Accounts
+         */
+        Utils.writeConsoleMessage("Creating Azure NetApp Files Account(s)...");
+        if (!config.getAccounts().isEmpty())
+        {
+            config.getAccounts().forEach(account -> createAccount(anfClient, account));
+        }
+        else
+        {
+            Utils.writeConsoleMessage("No ANF accounts defined within appsettings.json file. Exiting.");
+        }
+
+        /*
+          Creating Capacity Pools
+         */
+        Utils.writeConsoleMessage("Creating Capacity Pool(s)...");
+        for (ModelNetAppAccount modelAccount : config.getAccounts())
+        {
+            if (!modelAccount.getCapacityPools().isEmpty())
+            {
+                modelAccount.getCapacityPools().forEach(pool -> createCapacityPool(anfClient, modelAccount, pool));
```

Comment:
same thing. you should wait for ANF resource to return successful status

Reply:
Same thing, these are synchronous calls.

Reply:
even though these are synchronous calls, it will fail at some point, because it will be finishing up in the backend. As you noticed yesterday where I had to add the sleep because it failed.  As discussed, we need to get rid from the threadsleep and replace with the waiting functions that we already implemented.